single-query-track/process

```diff
@@ -14,10 +15,10 @@
 STATUS=$(
 # remove the StarExec timing information from each line of the solver output
     sed 's/^[0-9]*\.[0-9]*\/[0-9]*.[0-9]*\t//g' "$1" | \
-# filter out lines containing only sat, unsat, or unknown
-    egrep -e '^\s*(sat|unsat|unknown)\s*$' | \
+# remove empty lines and lines containing only success or a comment
+    egrep -v -e '^\s*(success|;.*)?\s*$' | \
 # the first answer wins
-    head -1 | egrep -o '(sat|unsat|unknown)')
+    head -1 | sed 's/^\s*\([a-z]\+\)\b.*/\1/')
```

Comment:
This won't work in OS X.  I'm not sure what's the right way around it though.  Either we don't care, or we use gsed in os X?